abs.yaml

```diff
@@ -1 +1,5 @@
 aggregate_branch: 3.12
```

Comment:
```suggestion
```

Reply:
I'll just delete the whole `abs.yaml` once we get jaxlib merged.